---
initial_pose:
  use: true
  bag1:
    position:
      x: -0.6174191236495972
      y: 0.40180954337120056
      z: 0.3308487832546234
    orientation:
      x: -0.005476718302816153
      y: 0.020145418122410774
      z: 0.6440925598144531
      w: 0.7646627426147461
  bag2:
      position:
        x: 0
        y: 0
        z: 0
      orientation:
        x: 0
        y: 0
        z: 0
        w: 1

